packages/featureflags/README.md

```diff
@@ -4,7 +4,7 @@ Feature flags
 If you need to hide some functionality or give access to it to a certain group of users, 
 then you need to use feature flags.
 
-***
+To use it you need to add `FeaturesMiddleware` to you `index.js`
```

Comment:
Specify, where to find it :)

Reply:
This package has only tree file and one of them is FeaturesMiddleware %) 
But ok, I'll explain it %))

Reply:
ahahah, my bad :) 

I was looking for GitHub diff only and it showed, that it has only has `...manager.js` file, that's I was struggling to find a  middleware 😄